lib/r10k/module/base.rb

```diff
@@ -60,8 +60,15 @@ def initialize(title, dirname, args, environment=nil)
     @environment = environment
     @overrides = args.delete(:overrides) || {}
     @spec_deletable = true
-    @exclude_spec = args.delete(:exclude_spec)
-    @exclude_spec = @overrides[:modules].delete(:exclude_spec) if @overrides.dig(:modules, :exclude_spec)
+    exclude_spec_from_args = @args.delete(:exclude_spec)
+    @exclude_spec = case
+                    when @overrides.dig(:modules, :exclude_spec) == false
```

Comment:
Why does it make sense to delete `:exclude_spec` from `@args`, but not from `@overrides`?